lib/Service/YAPServer.hpp

```diff
@@ -80,7 +78,8 @@ class YAPServer
         m_rxAttempts(0U),
         m_numberOfTxChannels(0U),
         m_numberOfRxChannels(0U),
-        m_numberOfPendingChannels(0U)
+        m_numberOfPendingChannels(0U),
+        m_rxCallbacks{nullptr}

```

Comment:
Wrong order of initializations, please use the same order as declared. Didn't you get warnings for the target build?

Reply:
No warnings on my side. Do you get the warnings?

Reply:
Yes ... compiling RemoteControlTarget.

Reply:
Should we consider adding -Werror to build flags, so the CI fails?

Reply:
Yes, we can do.